modules/apps/frontend-js/frontend-js-a11y-web/src/main/java/com/liferay/frontend/js/a11y/web/internal/configuration/A11yConfiguration.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.js.a11y.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Matuzalem Teles
+ */
+@ExtendedObjectClassDefinition(category = "infrastructure")
+@Meta.OCD(
+	description = "a11y-configuration-description",
+	id = "com.liferay.frontend.js.a11y.web.internal.configuration.A11yConfiguration",
+	localization = "content/Language", name = "a11y-configuration-name"
+)
+public @interface A11yConfiguration {
+
+	@Meta.AD(
+		deflt = "false", description = "enable-description",
+		name = "enable-name", required = false
+	)
+	public boolean enable();
```

Comment:
For now I'm adding this flag to disable a11y by default even with nodejs.node.env in development.

Reply:
Yeah, this makes sense to me as we may want to disable the tool under certain circumstances. I have to say that Chema has a stronger opinion as he'd prefer the tool to be always there.
A reasonable middle ground to me is to have this configuration hidden from the UI, like [this](https://github.com/liferay/liferay-portal/blob/51446053a22ecb2a1d8f6a971d36b2c4c68b6d23/modules/apps/asset/asset-auto-tagger-service/src/main/java/com/liferay/asset/auto/tagger/internal/configuration/AssetAutoTaggerGroupConfiguration.java#L25)

In case we go this way, we need another configuration which should be visible in the system settings to tweak the tool parameters

Reply:
Hmm I think I also prefer it not having the option, I just added it here as an attempt to send it before I integrate the Panel with the main component here just as a temp until we have everything in master and more stable.